models/public/rexnet-v1-x1.0/rexnet-v1-x1.0.md

```diff
@@ -0,0 +1,92 @@
+# rexnet-v1-x1.0
+
+## Use Case and High-Level Description
+
+ReXNet V1 x1.0 is image classification model pretrained on ImageNet dataset. Rank eXpansion Network (ReXNet) is a network derived from research to mitigate the representational bottleneck in a network.
+
+The model input is a blob that consists of a single image of "1x3x224x224" in RGB order.
+
+The model output is typical object classifier for the 1000 different classifications  matching with those in the ImageNet database.
+
+For details see [repository](https://github.com/clovaai/rexnet) and [paper](https://arxiv.org/pdf/2007.00992.pdf).
+
+## Specification
+
+| Metric           | Value          |
+| ---------------- | -------------- |
+| Type             | Classification |
+| GFLOPs           | 0.8325         |
+| MParams          | 4.7779         |
+| Source framework | PyTorch\*      |
+
+## Accuracy
+
+| Metric | Value  |
+| ------ | ------ |
+| Top 1  | 77.86% |
+| Top 5  | 93.87% |
+
+## Input
+
+### Original model
+
+Image, name - `input.1`,  shape - `1,3,224,224`, format is `B,C,H,W` where:
+
+- `B` - batch size
+- `C` - channel
+- `H` - height
+- `W` - width
+
+Channel order is `RGB`.
+Mean values - [123.675,116.28,103.53], scale values - [58.395,57.12,57.375].
+
+### Converted model
+
+Image, name - `input.1`,  shape - `1,3,224,224`, format is `B,C,H,W` where:
+
+- `B` - batch size
+- `C` - channel
+- `H` - height
+- `W` - width
+
+Channel order is `BGR`.
+
+## Output
+
+### Original model
+
+Object classifier according to ImageNet classes, name - `1424`,  shape - `1,1000`, output data format is `B,C` where:
+
+- `B` - batch size
+- `C` - Predicted probabilities for each class in  [0, 1] range
+
+### Converted model
+
+The converted model has the same parameters as the original model.
+
+## Legal Information
+
+The original model is distributed under the following
```

Comment:
original license does not contains license type inside, I suggest directly specify it like in the original repo

Reply:
Done